db/migrate/20190110142917_enable_content_security_policy_for_new_sites.rb

```diff
@@ -0,0 +1,25 @@
+class EnableContentSecurityPolicyForNewSites < ActiveRecord::Migration[5.2]
+  def up
+    return if Rails.env.test?
+    return if row_exists?
+
+    if instance_is_new?
+      execute "INSERT INTO site_settings(name, data_type, value, created_at, updated_at)
+               VALUES ('content_security_policy', 5, 't', now(), now())"
+    end
+  end
+
+  def down
+    # Don't undo, up method only enables CSP if row isn't already there and if instance is new
+  end
+
+  def row_exists?
+    return DB.query("SELECT 1 AS one FROM site_settings where name='content_security_policy'").present?
+  end
+
+  def instance_is_new?
+    post = DB.query("SELECT created_at FROM posts WHERE id=1")
```

Comment:
What if the post with id=1 has been deleted for some reason? I think it would be better to do something like (untested)

```SQL
SELECT created_at FROM posts ORDER BY created_at ASC LIMIT 1
```